tests/TestCase/ToolbarServiceTest.php

```diff
@@ -73,6 +73,25 @@ public function testLoadPanels()
         $this->assertGreaterThan(1, $this->events->listeners('Controller.shutdown'));
         $this->assertInstanceOf('DebugKit\Panel\SqlLogPanel', $bar->panel('SqlLog'));
     }
+    
+    /**
+     * Test disabling panels.
+     *
+     * @return void
+     */
+    public function testDisablePanels()
+    {
+        $bar = new ToolbarService($this->events, ['panels' => [
+            'DebugKit.SqlLog' => false,
+            'DebugKit.Cache' => true,
+            'DebugKit.Session'
```

Comment:
All panels are enabled by default (due to the `$_defaultConfig`). So you explicitly need to disable panels you don't want. Only custom panels need to be included/enabled (as before).

When a panel is contained with a numeric key **and** set to `false` it will be loaded, but this can only happen if the user includes something like `'panels' => ['Test', 'Test' => false]` in their `config/app.php` which *should* not happen...